### ISSUE TITLE
iPad missing from Targeted Device Family

### ISSUE THREAD

**Issue by skypanther** [mokesmokes/titanium-ios-facebook#21]
I was getting errors that the armv7 architecture was missing from the module. Looking at your Xcode project, I found that the source of the problem was actually the missing iPad target. Adding that and rebuilding the module, it now works fine on my iPad2. 

This relates to version 3.21.10

**Comment by mokesmokes** [comment]
Weird.... I didn't touch this parameter at all. I just tried to create a default module project through Studio, using 3.4.1.GA, then opened the project in Xcode. Looking at Targeted Device Family I indeed see iPhone and not iPhone/iPad. So looks like a bug with Titanium's default settings for modules.
The modules should definitely be iPhone/iPad by default, with the actual effective setting at the project level. Agree? Will you open a JIRA ticket on this?

**Comment by skypanther** [comment]
You bet. I'll check into the module settings in the default project. They may have had a reason for iPhone only, in which case we should document that clearly.

**Comment by mokesmokes** [comment]
I'm also somewhat surprised that that parameter has any effect for modules. I would expect it makes a difference only when building the complete project.

**Comment by skypanther** [comment]
This was the error I got when using your 3.21.10 version building my app to my iPad2

```shell
[ERROR] Script Error Couldn't find module: alloy/controllers/mycontroller for architecture: armv7
```

The app had been working fine with 3.4.1 and an older version of your FB module. I'm trying to get it 64-bit ready so I tried building with both 3.5.0.RC and RC2. Then, I eliminated all modules and added them back in one by one. The error returned when I added in your module at the point where I called its methods. 

First I checked to see if you included the armv7 arch at all (you do). Then I noticed the targeted devices option. I figured it was worth a try and added iPad to the targeted devices, ran build.py, copied the .a file over the one in the module folder, built to my iPad again and the error went away. I am certainly no modules expert! I hope those are valid steps.

**Comment by mokesmokes** [comment]
@skypanther do your other modules have iPad as a targeted device?

**Comment by skypanther** [comment]
@mokesmokes hmm, no, it looks like they do not. 

I eliminated all modules from my project, then added them back in one by one. Adding this module caused the error to reappear. So, seems like it's the cause. But, maybe it was something else in your project's config that was the actual root cause. I rebuilt the project after changing the target device list. But in doing so, my Xcode might have been "helpful" and changed something else that was the actual reason for the issue.

I guess if no one else is running into this, go ahead and close the issue. I'm up and running using my build version.
